_posts/2017/2017-02-20-continuous-documentation-with-ansible.md

```diff
@@ -0,0 +1,207 @@
+---
+layout: post
+title: "Continuous Documentation with Ansible"
+date: "2017-02-20 06:39:20"
```

Comment:
Depending on when you can include the feedback, we can either publish the first part of the post on 14th or 16th of March.